### ISSUE TITLE
Ship sale and rental prices and locations

### ISSUE THREAD

**Issue by cintaracorp** [richardthombs/scunpacked#23]
Are the ship sale and rental prices and locations in the data somewhere?

**Comment by richardthombs** [comment]
Yep, everything on scunpacked comes straight from the data. If you can read code, check this file out for info: [https://github.com/richardthombs/scunpacked/blob/master/Loader/ShopLoader.cs](https://github.com/richardthombs/scunpacked/blob/master/Loader/ShopLoader.cs)

**Comment by cintaracorp** [comment]
@richardthombs Thanks, found them now!

**Comment by cintaracorp** [comment]
@richardthombs Looking at the results, I have problems figuring out the ship rental prices. I suspect those that are listed as "Buy" are really rentals. Do you happen to know the formula to go to rental prices from the numbers listed here? Otherwise I will just go in-game to compare the rental terminals to your numbers and see if I can figure something out! :-)

**Comment by cintaracorp** [comment]
@richardthombs OK, I checked. Unfortunately, there seems to be some arbitrary choice between a 50x and 33x price ratio for 1-day rentals, as well as a couple of weird "almost-there" ratios, possibly mistypes when copying the data.

Ship rentals: | "Sell" | 1 day | Ratio
-- | -- | -- | --
Avenger | 785600 | 15712 | 50
Warden Lorville | 3387800 | 101634 | 33.33333333
Arrow | 972300 | 19446 | 50
Nox | 349200 | 10476 | 33.33333333
Mustang Alpha | 251400 | 5028 | 50
Mustang Gamma | 627,500.00 | 18825 | 33.33333333
Cutlass Black | 1,385,300.00 | 27706 | 50
Dragonfly | 272,700.00 | 5454 | 50
Freelancer | 1,697,600.00 | 33952 | 50
Prospector | 2,061,000.00 | 41220 | 50
Razor | 1,761,200.00 | 52836 | 33.33333333
300i | 767,200.00 | 23016 | 33.33333333
600i | 9,475,100.00 | 284253 | 33.33333333
M50 | 1,193,800.00 | 35814 | 33.33333333
Aurora ES | 124,100.00 | 2848 | 43.5744382
Andromeda | 3,548,000.00 | 70960 | 50
Cyclone | 27,599.08 | 551 | 50.08907441

**Comment by richardthombs** [comment]
Hey @cintaracorp I've been busy with COVID child minding and I've let this issue slide.

So the purchase prices are correct, but rental prices are not?

**Comment by cintaracorp** [comment]
Well, the one-day rentals was a specific fraction of the price you had in "Buys" for the rental shops. However, those numbers were during the 3.9 Evocati phase, so they might very well have changed now that 3.10.1 is out. However, using those ratios is probably better than nothing, I guess it is more likely that they tweak the base price than the ratio.
